Makefile

```diff
@@ -23,6 +23,22 @@ help: ## display help for this makefile
 rm-pycache: ## remove all __pycache__ files (run if encountering issues with pycharm debugger (containers exiting prematurely))
 	find . -name '__pycache__' | xargs rm -rf
 
+### Connecting to remote dbs #########################################
+remotedb: # Get a psql console on a remote db (from OSX only!)
+	export ENV=$${ENV:=rdev}; \
+	export config=$$(aws --profile $(AWS_DEV_PROFILE) secretsmanager get-secret-value --secret-id $${ENV}/aspen-config | jq -r .SecretString ); \
+	export DB_URI=$$(jq -r '"postgresql://\(.DB.admin_username):\(.DB.admin_password)@127.0.0.1:5556/$(STACK)"' <<< $$config); \
+	ssh -f -o ExitOnForwardFailure=yes -L 5556:$$(jq -r .DB.address <<< $$config):5432 $$(jq -r .bastion_host <<< $$config) sleep 10; \
+	psql $${DB_URI}
+
+remoteconsole: .env.ecr # Get a python console on a remote db (from OSX only!)
+	export ENV=$${ENV:=rdev}; \
+	export config=$$(aws --profile $(AWS_DEV_PROFILE) secretsmanager get-secret-value --secret-id $${ENV}/aspen-config | jq -r .SecretString ); \
+	export OSX_IP=$$(ipconfig getifaddr en0 || ipconfig getifaddr en1); \
+	export DB_URI=$$(jq -r '"postgresql://\(.DB.admin_username):\(.DB.admin_password)@'$${OSX_IP}':5555/$(STACK)"' <<< $$config); \
+	ssh -f -o ExitOnForwardFailure=yes -L $${OSX_IP}:5555:$$(jq -r .DB.address <<< $$config):5432 $$(jq -r .bastion_host <<< $$config) sleep 10; \
```

Comment:
We're binding the remote postgres server's port 5432 to port 5555 on our primary OSX IP address so that Docker can route to it (it can't reach ports bound to 127.0.0.1 on OSX!)